agent/src/dbus/softwarecontaineragent_dbus_stub.h

```diff
@@ -91,14 +92,23 @@ class SoftwareContainerAgent {
            const Glib::ustring& property_name,
            const Glib::VariantBase& value);
 
-private:
-bool emitSignal(const std::string& propName, Glib::VariantBase& value);
+    /*
+     * We emit this signal when we have a lost/acquired a name on dbus,
```

Comment:
It's a list of three signals so "this" doesn't feel right to me. Just a nitpicky comment, but might as well do it right. ;)